pkg/test/common/utils.go

```diff
@@ -177,3 +180,16 @@ func CustomRedirectPolicy(noOfRedirect int) resty.RedirectPolicy {
 		return nil
 	})
 }
+
+func GenerateRandomString() string {
```

Comment:
Given the character constraints, maybe have different functions such as GenerateRandomRepoName(), GenerateRandomUsername(), GenerateRandomPassword(). Just a suggestion.